dist/docker/debian/build_docker.sh

```diff
@@ -69,9 +69,6 @@ packages=(
     "build/dist/$config/debian/$product-conf_$version-$release-1_$arch.deb"
     "build/dist/$config/debian/$product-kernel-conf_$version-$release-1_$arch.deb"
```

Comment:
why do we need kernel configuration inside the docker image? What does it do?

Reply:
It's one of Scylla-server dependency